src/containers/support/FAQ.tsx

```diff
@@ -11,9 +11,8 @@ const generalQuestions: SupportQuestion[] = [
         No - this is a support page for software developers utilizing the Veterans Affairs (VA)
         Application Programming Interface (API). If you are a Veteran seeking assistance, please
         visit the <a href="http://www.va.gov">U.S. Department of Veterans Affairs website</a> to
-        access and manage your VA benefits and health care. There are also helpful reference links
-        and Q&amp;A at the VA Inquiry Routing &amp; Information System{' '}
-        <a href="https://iris.custhelp.va.gov/app/answers/list">(IRIS)</a>.
+        access and manage your VA benefits and health care. You may also ask your question at 
+        Ask VA{' '} <a href="https://ask.va.gov/">(AVA)</a>.
```

Comment:
Linting rules seem to have tripped up your PR. This should resolve it.

```suggestion
        access and manage your VA benefits and health care. You may also ask your question at Ask
        VA <a href="https://ask.va.gov/">(AVA)</a>.
```